torchhd/datasets/dataset.py

```diff
@@ -0,0 +1,331 @@
+import os
+import os.path
+from typing import Callable, Optional, Tuple, List
+import torch
+from torch import Tensor
+from torch.utils import data
+import pandas as pd
+import tarfile
+import numpy as np
+import torchhd
+
+
+from .utils import download_file_from_google_drive
+
+
+# Function that iterates via a given collection of datasets
+def UCIDatasetCollection(dataset_collection, root, download):
```

Comment:
Okay, I thought your previous comment was going towards more flexibility so that, e.g., a subset of the collection could be used since the function is a separate primitive. So dataset_collection variable was added with this in mind.

Reply:
Ah I see, I guess you could still skip datasets by not training on them?

Reply:
The best way I found to make it working was to add a constant UCI_DATASET_COLLECTION (used capitals to follow the convention for constants) into dataset.py and the refer to torchhd.datasets.UCI_DATASET_COLLECTION in the example code.